configs/messages.go

```diff
@@ -17,47 +17,48 @@ package configs
-	PrintingFile                    = "File %s:"
-	SupportedClients                = "Supported clients of type %s: %v"
-	ConfigClientsMsg                = "Provided clients of type %s in configuration file: %v"
-	RunningDockerCompose            = "Running docker-compose script"
-	Component                       = "component"
-	RunningCommand                  = "Running command: %s"
-	ConfigFileName                  = ".sedge"
-	UnableToProceed                 = "Unable to proceed. Please check the logs for more details"
-	DefaultDockerComposeScriptName  = "docker-compose.yml"
-	CheckingDockerEngine            = "Checking if docker engine is on"
-	DepositCLIDockerImageUrl        = "nethermindeth/staking-deposit-cli" //"github.com/ethereum/staking-deposit-cli"
-	DepositCLIDockerImageName       = "nethermindeth/staking-deposit-cli" //"deposit-cli:local"
-	GeneratingKeystores             = "Generating keystores..."
-	GeneratingKeystoresLegacy       = "Generating keystore folder"
-	KeystoresGenerated              = "Keystores generated."
-	GeneratingDepositData           = "Generating deposit data..."
-	DepositDataGenerated            = "Deposit data generated."
-	KeysFoundAt                     = "If everything went well, your keys can be found at: %s"
-	ImageNotFoundBuilding           = "Image %s not found, building it"
-	ImageNotFoundPulling            = "Image %s not found, pulling it"
-	ExecutionDefaultDataDir         = "./execution-data"
-	ConsensusDefaultDataDir         = "./consensus-data"
-	ValidatorDefaultDataDir         = "./validator-data"
-	KeystoreDefaultDataDir          = "./keystore"
-	ReviewKeystorePath              = "In case you used custom paths for the 'cli' or the 'keys' commands, please review if the keystore path in the generated .env file points to the generated keystore folder (the .env key should be KEYSTORE_DIR). If not, change the path in the .env file to the correct one."
-	NodesSynced                     = "Execution and Consensus clients are synced, proceeding to start validator node"
-	RemoteNodeNeeded                = `
+	CheckingDependencies           = "Checking dependencies: %s"
+	DependenciesPending            = "pending dependencies: %s"
+	DependenciesOK                 = "All dependencies are installed on host machine"
+	GeneratingDockerComposeScript  = "Generating docker-compose script for current selection of clients"
+	GeneratingEnvFile              = "Generating environment file for current selection of clients"
+	Exiting                        = "Exiting..."
+	InstructionsFor                = "Instructions for %s"
+	OSNotSupported                 = "installation not supported for %s"
+	ProvideClients                 = "Please provide both execution client and consensus client"
+	CreatedFile                    = "Created file %s"
+	DefaultSedgeDataFolderName     = "sedge-data"
+	DefaultSedgeDataPath           = "./" + DefaultSedgeDataFolderName
```

Comment:
This was wrong before. We shouldn't use `./` in any constant. Instead of that we should get the current WD and use `filepath.Join()` to create the path, as this is cross platform.

Reply:
In that case we would use something like this to refer that path.
Let `cwd` be the current working directory:
```
path = filepath.Join(cwd, configs.DefaultSedgeDataFolderName)
```

Reply:
Done here [3dea928](https://github.com/NethermindEth/sedge/pull/159/commits/3dea928fe423eeeb9cf523b6fbcb50a0b5d2ffc7)